shared/constants/types/fs.js

```diff
@@ -119,6 +136,10 @@ export const pathIsNonTeamTLFList = (p: Path): boolean => {
   return str === '/keybase/private' || str === '/keybase/public'
 }
 
+const localSep = isWindows ? '\\' : '/'
+
+export const localPathConcat = (p: LocalPath, s: string): LocalPath => p + localSep + s
+
```

Comment:
path.join() should do the right thing here I think?

Reply:
I think the issue is `path` isn't present in react-native. I even tried pulling in selected functions from the node `path` package but Nojima thought we should make it simple for now, until we really need to solve those edge cases.